src/makeInfinityAdder.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  function adder(n) {
```

Comment:
It's a good practice to apply more clear names
```suggestion
  function adder(number) {
```